src/main/java/com/github/dockerjava/api/model/Statistics.java

```diff
@@ -19,6 +21,17 @@
     @JsonProperty("read")
     private String read;
 
+    /**
+     * @since Docker Remote API 1.21
+     */
+    @CheckForNull
+    @JsonProperty("networks")
+    private Map<String, Object> networksStats;
```

Comment:
Variables are normally named like JSON fields in docker-java. But what about getter? getNetwork()/getNetworks() should return an dedicated object instead of a map, right?